ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/prater.yaml

```diff
@@ -71,6 +74,8 @@ EJECTION_BALANCE: 16000000000
 MIN_PER_EPOCH_CHURN_LIMIT: 4
 # 2**16 (= 65,536)
 CHURN_LIMIT_QUOTIENT: 65536
+# [New in Deneb:EIP7514] 2**3 (= 8)
+MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 8
 
 
 # Fork choice
```

Comment:
Are we missing updating PROPOSER_SCORE_BOOST to 40?

Reply:
Yeah, think so. Changed it. But this is not DENEB specific, so it's weird it hasn't been changed.